src/game/items/CItem.cpp

```diff
@@ -2247,12 +2247,6 @@ void CItem::r_WriteMore1(CSString & sVal)
     ADDTOCALLSTACK("CItem::r_WriteMore1");
     // do special processing to represent this.
 
-	if (IsTypeSpellbook())
-	{
-		sVal.FormatHex(m_itNormal.m_more1);
```

Comment:
Can i ask why was this and the similar function for more2 removed? For spellbooks, more1 and more2 are bitmasks holding the known spells, it's more readable as hex than as decimal

Reply:
I mean, there's FormatULLHex

Reply:
because it's already done it automatically in this case, there is a Switch under the check and default return is sVal.FormatHex(m_itNormal.m_more1); so second check for same thing seems unnecessary for me

        default:
            if (CResourceIDBase::IsValidResource(m_itNormal.m_more1))
                sVal = g_Cfg.ResourceGetName(CResourceID(m_itNormal.m_more1, 0));
            else
                sVal.FormatHex(m_itNormal.m_more1);
            return;

these are the lines I mentioned.

Reply:
Oh i see, good one!